ee/clickhouse/views/experiments.py

```diff
@@ -250,11 +250,14 @@ def create(self, validated_data: dict, *args: Any, **kwargs: Any) -> Experiment:
             feature_flag_serializer.is_valid(raise_exception=True)
             feature_flag = feature_flag_serializer.save()
 
-        if not validated_data.get("stats_config"):
+        # Ensure stats_config has a method set, preserving any other fields passed from frontend
+        stats_config = validated_data.get("stats_config", {})
```

Comment:
This is more a question than a comment. Wouldn't it make more sense to deprecate `stats_config` and maybe create a new `metadata` to store these random, sometimes temporal, things?